README.md

```diff
@@ -8,40 +8,48 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 
 ```bash
 npm install
-npm --prefix ./scripts install
```

Comment:
the build:website script would fail if we don't run this command too

Reply:
In the build script in scripts/build-wesbite.sh, I saw it is the first command that's there. Should we still add it back?

<img width="1079" alt="Screenshot 2024-05-05 at 11 31 19 AM" src="https://github.com/fastify/website/assets/47499826/e71d3788-c268-458c-ab0d-ae90e1698910">

Reply:
no, I totally forgot it - thanks

LGTM